src/page/HomePage.js

```diff
@@ -0,0 +1,27 @@
+const HomePage = () => (
+  <div>
+    <h2 className="text-4xl py-16 px-4">Welcome To Our Page</h2>
+    <p className="py-8 px-4">
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+      Eaque temporibus voluptates perferendis fugiat corporis magni?
+      perspiciatis perferendis vero nam quibusdam ea, iure optio veritatis reiciendis
+      Reprehenderit, accusamus neque? Repellat exercitationem excepturi illo inventore
+      Lorem ipsum dolor sit amet, consectetur adipisicing eliure, illum consequatur esse
+      Omnis velit vel quam architecto voluptas, cum nostrum neqminus facere
+      Debitis porro praesentium placeat laudantium aperisapiente id magnam perspiciatis i
+      Repellat rem consequatur ducimus expedita magnam ex! Eanam dignissimos? Dolorem deser
+    </p>
+    <p className="py-8 px-4">
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+      Eaque temporibus voluptates perferendis fugiat corporis magni?
+      perspiciatis perferendis vero nam quibusdam ea, iure optio veritatis reiciendis
+      Reprehenderit, accusamus neque? Repellat exercitationem excepturi illo inventore
+      Lorem ipsum dolor sit amet, consectetur adipisicing eliure, illum consequatur esse
+      Omnis velit vel quam architecto voluptas, cum nostrum neqminus facere
+      Debitis porro praesentium placeat laudantium aperisapiente id magnam perspiciatis i
+      Repellat rem consequatur ducimus expedita magnam ex! Eanam dignissimos? Dolorem deser
+    </p>
```

Comment:
- Optional
Consider giving this section a bit more padding on the x-axis. Right now it is looking too much cramped and I am sure you would like it with a bit more padding.⬇️
![image](https://user-images.githubusercontent.com/33268927/212671255-22a2e58d-be94-409c-be74-9d716248a78c.png)

Reply:
Ya by giving more padding it will look more beautiful thanks for your suggestion I will work on it.